docs/guides/BUILDING.md

```diff
@@ -131,21 +131,27 @@ permanently, you need to make the following changes:
 
 ## Installing ZAP tool
 
-`bootstrap.sh` will download a compatible ZAP tool version and set it up in
-`$PATH`. If you want to install or use a different version of the tool, you may
-download one from the ZAP project's
-[Releases](https://github.com/project-chip/zap/releases) page.
+For platforms defined in [`scripts/setup/zap.json`](/scripts/setup/zap.json),
+`bootstrap.sh` will download a compatible ZAP tool version from CIPD and set it
+up in `$PATH`.
 
-### Linux ARM
+ZAP releases are copied to CIPD by an automated bot. You can check if a release
+was copied by looking at tags created at
+[here](https://chrome-infra-packages.appspot.com/p/fuchsia/third_party/zap) in
```

Comment:
Please never use "here" as link text.  If you have no better idea, just use the whole URL.  But probably we can do better here.